assets/content/cookbook/Macros/compare-anon-structures.md

```diff
@@ -0,0 +1,131 @@
+# Compare Anonymous Structures and Objects
+
+## Why use a macro
+
+Sometimes it can be useful to compare two anonymous structures or objects
+ by their properties, even if they're not the same.
```

Comment:
Also it might be better to talk about "fields" instead of properties, as the latter means something else in Haxe.